src/components/ModalDialog/ModalDialog.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import './modal-dialog.scss';
+
+const modalRoot = document.createElement('div');
```

Comment:
Please try to avoid using document object and implement with another approach

Reply:
@NikitaUstsiushenka do you mean use effect?